src/component/admin/groups/CreateGroup/CreateGroup.tsx

```diff
@@ -43,8 +44,12 @@ export const CreateGroup = () => {
                 confetti: true,
                 type: 'success',
             });
-        } catch (error: unknown) {
-            setToastApiError(formatUnknownError(error));
+        } catch (error: any) {
+            if (error.body.name === 'NameExistsError') {
+                setErrors(prev => ({ ...prev, name: error.message }));
```

Comment:
I think setErrors is only for field validations on the form. That is the only field that is validated.

Reply:
I think only places that uniqueness is validated are using `/validate` endpoint in the middle, which I think is too much for this component.

Reply:
I see your point. 
Should we at least add `NameExistsError` as a const / enum somewhere that can be imported instead of using the literal string?
Or maybe create a method similar to `formatUnknownError` that returns both error type and message from an unknown error to clean this up a bit and be a bit more standardized.

Reply:
Removed for now, because could not solve it prettily.